stake-pool/cli/src/output.rs

```diff
@@ -0,0 +1,435 @@
+                Sol(stake_account.validator_lamports),
+                stake_account.validator_last_update_epoch,
+            )?;
+        }
+        writeln!(
+            f,
+            "Total Pool Stake: {} {}",
+            Sol(*&self.total_lamports),
+            &self.flag,
+        )?;
+        writeln!(
+            f,
+            "Total Pool Tokens: {}",
+            &self.total_pool_tokens,
+        )?;
+        writeln!(
+            f,
+            "Current Number of Validators: {}",
+            &self.current_number_of_validators,
+        )?;
+        writeln!(
+            f,
+            "Max Number of Validators: {}",
+            &self.max_number_of_validators,
+        )?;
+        Ok(())
+    }
+}
+
+impl QuietDisplay for CliStakePoolDetails {}
+impl VerboseDisplay for CliStakePoolDetails {
+    fn write_str(&self, w: &mut dyn Write) -> Result {
+        writeln!(w, "")?;
+        writeln!(w, "Stake Accounts")?;
+        writeln!(w, "--------------")?;
+        writeln!(
+            w,
+            "Reserve Account: {}\tAvailable Balance: {}",
+            &self.reserve_stake_account_address,
+            Sol(&self.reserve_stake_lamports - &self.minimum_reserve_stake_balance),
+        )?;
+        for stake_account in &self.stake_accounts {
+            writeln!(
+                w,
+                "Vote Account: {}\tStake Account: {}\tActive Balance: {}\tTransient Stake Account: {}\tTransient Balance: {}\tLast Update Epoch: {}{}",
+                stake_account.vote_account_address,
+                stake_account.stake_account_address,
+                Sol(stake_account.validator_active_stake_lamports),
+                stake_account.validator_transient_stake_account_address,
+                Sol(stake_account.validator_transient_stake_lamports),
+                stake_account.validator_last_update_epoch,
+                stake_account.flag,
+            )?;
+        }
+        writeln!(
+            w,
+            "Total Pool Stake: {} {}",
+            Sol(*&self.total_lamports),
+            &self.flag,
+        )?;
+        writeln!(
+            w,
+            "Total Pool Tokens: {}",
+            &self.total_pool_tokens,
+        )?;
+        writeln!(
+            w,
+            "Current Number of Validators: {}",
+            &self.current_number_of_validators,
+        )?;
+        writeln!(
+            w,
+            "Max Number of Validators: {}",
+            &self.max_number_of_validators,
+        )?;
+        Ok(())
+    }
+}
+
+#[derive(Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub(crate) struct CliStakePoolStakeAccountInfo {
+    pub vote_account_address: String,
+    pub stake_account_address: String,
+    pub validator_active_stake_lamports: u64,
+    pub validator_last_update_epoch: u64,
+    pub validator_lamports: u64,
+    pub validator_transient_stake_account_address: String,
+    pub validator_transient_stake_lamports: u64,
+    pub flag: String,
```

Comment:
```suggestion
    #[serde(skip)]
    pub update_required: String,
```

Reply:
Ok good idea but as boolean I guess

Reply:
Oops, yes, edited

Reply:
I don't skip serde. I think it could be a usefull info in json format.

Reply:
Fair enough. I'll let @joncinque holler if he has any objections.

Reply:
that works for me!